fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubberTest.java

```diff
@@ -335,11 +340,37 @@ void testScrubberLimitsAlternatingLegacy(boolean legacyInit) {
             }
             assertEquals(0, resDangling);
             assertEquals(0, resMissing);
+
+            final ScrubbersMissingRanges ranges = getScrubbersMissingRange(tgtIndex);
+            assertNotEquals(lastRanges.indexes, ranges.indexes);
+            assertNotEquals(lastRanges.records, ranges.records);
+            lastRanges = ranges;
         }
         assertEquals(numRecords * 2, timer.getCount(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RECORDS_SCANNED));
         assertEquals(0, timer.getCount(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RECORDS_INDEXED));
         assertEquals(0, timer.getCount(FDBStoreTimer.Counts.INDEX_SCRUBBER_MISSING_ENTRIES));
         assertEquals(0, timer.getCount(FDBStoreTimer.Counts.INDEX_SCRUBBER_DANGLING_ENTRIES));
+        assertTrue(lastRanges.indexes == null || RangeSet.isFirstKey(lastRanges.indexes.begin));
+        assertTrue(lastRanges.records == null || RangeSet.isFirstKey(lastRanges.records.begin));
```

Comment:
Why would it be the first key?

Reply:
The RangeSet may be cleared by https://github.com/jjezra/fdb-record-layer/blob/11e161c9f7f252fcd3046d9dfa9ff2eded232b00/fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexScrubbing.java#L162

Here the two cursors (records, index entries) behave a differently,. I am not sure why.

Reply:
It would be good to figure out why these behave differently. I can't think of any reason they should